### DISCUSSION TITLE
Hello world window is blurry on linux mint

### DISCUSSION THREAD

**Discussion by inshua** [AvaloniaUI/Avalonia#13589] [Q&A]
## Describe the bug
See the snapshot
![image](https://github.com/AvaloniaUI/Avalonia/assets/2752067/2607f72d-35da-4282-bae2-0ebadca84c10)

## To Reproduce
Just create a project and run 

## Environment 

Linux Mint 21.2 Cinnamon
- Avalonia-Version:  11.0.2

## Additional context
I've set the background to solid color, but it still failed
```xaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="AvaloniaApplication3.MainWindow"
        Title="AvaloniaApplication3" Background="Red">
    Welcome to Avalonia!
</Window>
```

**Comment by timunie** [comment]
Do you have all Avalonia versions the same? Please check carefully. Any Styles added?

**Comment by Coloryr** [comment]
Running in virtual machine?

**Comment by inshua** [comment]
> Do you have all Avalonia versions the same? Please check carefully. Any Styles added?

I only have 11.0.2, the default Theme is FluentTheme, and I've tried no theme, SimpleTheme, the problem still exists.

.net core 7.0.113

**Comment by inshua** [comment]
> Running in virtual machine?

No, it's a physical machine, my working computer.

**Comment by Coloryr** [comment]
use 11.0.5?

**Comment by inshua** [comment]
> use 11.0.5?

Thank you, I've tried 11.0.5, the problem still exists.
![image](https://github.com/AvaloniaUI/Avalonia/assets/2752067/1378e307-ad71-43e1-86bc-4085f82b253a)

**Comment by Coloryr** [comment]
is look like opengl or your graphics card drive issus.

**Comment by inshua** [comment]
> is look like opengl or your graphics card drive issus.

I have 2 nvidia cards, one is 2080 and another is 4090.

+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.129.03             Driver Version: 535.129.03   CUDA Version: 12.2     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 4090        Off | 00000000:0B:00.0 Off |                  Off |
|  0%   41C    P8              26W / 450W |     14MiB / 24564MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   1  NVIDIA GeForce RTX 2080 Ti     Off | 00000000:43:00.0  On |                  N/A |
| 41%   50C    P0              56W / 250W |    850MiB / 11264MiB |      1%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+


Do you have some opengl test util to confirm it?

**Comment by Coloryr** [comment]
https://github.com/AvaloniaUI/Avalonia/blob/master/samples/ControlCatalog/Pages/OpenGlPage.xaml.cs
Clone the Avalonia and build it.
run
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/33ae1ce5-1a2a-4e1b-a8dc-eefb80dee0cc)

**Comment by inshua** [comment]
> https://github.com/AvaloniaUI/Avalonia/blob/master/samples/ControlCatalog/Pages/OpenGlPage.xaml.cs Clone the Avalonia and build it. run ![image](https://user-images.githubusercontent.com/26276037/282394566-33ae1ce5-1a2a-4e1b-a8dc-eefb80dee0cc.png)

I've tried to add OpenGlPage, but the OpenGLPage is blurry too since the window is blurry. 
And to build it on my linux failed too. no namespace ILRepacking and Numerge.

**Comment by Coloryr** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/2ca6f23a-28a1-4842-9a4e-f95533683dd2)
in my mint vm.

**Comment by inshua** [comment]
> ![image](https://user-images.githubusercontent.com/26276037/282401418-2ca6f23a-28a1-4842-9a4e-f95533683dd2.png) in my mint vm.

well, hope there is some util to diagnose it.

**Comment by timunie** [comment]
@inshua would be helpful maybe if you can upload your project here for others to run on their machines.

**Comment by inshua** [comment]
> @inshua would be helpful maybe if you can upload your project here for others to run on their machines.

Thank you, I am a newbie for Avalonia, and it's only the Hello world project, I created the project in Rider. But now I created a new blank app and it works fine.

I think the problem come from Rider, but I have no idea about it.

![image](https://github.com/AvaloniaUI/Avalonia/assets/2752067/10e0dfc6-c78a-4909-9dce-70eba1ae6fd8)

Same binary, one starts with Rider, one starts from shell.

**Comment by timunie** [comment]
Don't know Rider on Linux. Probably the debugger has an issue or was blocking something? However, I'll convert this issue into a discussion as it seems not to be an Avalonia issue in the end.

**Reply by thevortexcloud** [reply]
I personally have never had an issue using Rider on Linux with Avalonia baremetal. If anything it works better than Visual Studio does on Windows. But I have also never tried it with an Nvidia GPU.

In any case one thing I can suggest is to try to update your graphics driver/switch to a different one. Nvidia's proprietary drivers on Linux are known to break seemingly at random for various reasons (generally some system update which they are not compatible with). Unfortunately the open source driver is still fairly early in development, but it may work better. If you can, try an intel/AMD GPU too. But all of this may not end up helping anyway.
